ga4gh/server/frontend.py

```diff
@@ -1080,12 +1075,11 @@ def gateway_logout():
     End flask login sessions. Tyk will handle remote keycloak session
     :return: redirect to the keycloak login
     """
-    # response = flask.redirect(_generate_base_url()+'/login_oidc')
+
     if not flask.request.cookies.get("session_id"):
         raise exceptions.NotAuthenticatedException
 
-    data = {"redirect": _generate_logout_url()}
-    response = flask.Response(json.dumps(data))
+    response = flask.Response({})
```

Comment:
Previously, `flask.Response` takes in a json string (dumped from `data`), but now it's taking in an empty object, is this going to be a problem?

Reply:
Oh I see, that's fine then!